packages/stacks/src/validation/transaction-validation.ts

```diff
@@ -0,0 +1,27 @@
+import { ChainId } from '@leather.io/models';
+
+import {
+  isValidAddressChain,
+  isValidStacksAddress,
+  validatePayerNotRecipient,
+} from './address-validation';
+import { StacksError } from './stacks-error';
+
+export function isValidStacksTransaction(
+  senderAddress: string,
+  recipientAddress: string,
+  chainId: ChainId
+) {
+  if (!isValidStacksAddress(senderAddress) || !isValidStacksAddress(recipientAddress)) {
```

Comment:
Right now I'm just checking addresses are correct but there is more work to come validating amounts, compliance etc.